.github/workflows/linux.yml

```diff
@@ -312,6 +312,22 @@ jobs:
           name: hal-compile-quick-4
           path: test/all/log
 
+  build-protobuf-example:
+    runs-on: ubuntu-20.04
+    container:
+      image: ghcr.io/modm-ext/modm-build-cortex-m:2022-03-28
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Checkout code and update modm tools
+        run: |
+          (git submodule sync && git submodule update --init --jobs 8) & pip3 install --upgrade --upgrade-strategy=eager modm & wait
+      - name: Compile protobuf example
+        if: always()
+        run: |
+          (cd examples && ../tools/scripts/examples_compile.py protobuf)
+
+
```

Comment:
Please add the example to the nucleo_f429zi folder and remove the extra job (it takes a few minutes to launch a new container for this, way overkill for one example).